actions/update_field_value.py

```diff
@@ -0,0 +1,14 @@
+from lib.base import BaseJiraAction
+
+__all__ = [
+    'UpdateFieldValue'
+]
+
+
+class AddFieldValue(BaseJiraAction):
```

Comment:
This should make the linter happy here:
```suggestion
class UpdateFieldValue(BaseJiraAction):
```